index.js

```diff
@@ -1,11 +1,37 @@
-export default function uniqueRandom(minimum, maximum) {
+function filledArray(minimum, maximum) {
+	const array = [];
+	for (let i = minimum; i < maximum + 1; i++) {
+		array.push(i);
+	}
+
+	return array;
+}
+
+export default function uniqueRandom(minimum, maximum, options) {
+	const defaultArray = filledArray(minimum, maximum);
```

Comment:
Logic for `noOverlap` and the default behaviour can be seperated with a high-level if statement because of how different their implementations are.